### PR TITLE
fixed $defaults according to #2430

### PR DESCRIPTION
**Pull request by ThomasWollmann**

### CODE REVIEW

**Comment by jmchilton** [comment]
Seems like a good idea, thanks!

**Comment by jmchilton** [comment]
@galaxybot test this

**Comment by bgruening** [comment]
@jmchilton I think the failing test is unrelated: `ERROR: Step ‘Publish JUnit test result report’ failed: No test report files were found. Configuration error?`

File: lib/galaxy/tools/deps/containers.py [hunk 1]
```diff
@@ -303,6 +306,9 @@ def add_var(name, value):
         else:
             defaults = "$galaxy_root:ro,$tool_directory:ro,$working_directory:rw,$default_file_path:rw"
 
+        if self.app_info.library_import_dir:
+            defaults = defaults + ",$library_import_dir:ro"
```
Comment (remimarenco) [inline lib/galaxy/tools/deps/containers.py]: Could be also:

``` python
defaults += ",$library_import_dir:ro"
```

But I guess it depends on how much you want your code to be obvious :)

**Comment by bgruening** [comment]
Thanks @ThomasWollmann!

**Pull request merged**